gp/node.py

```diff
@@ -44,33 +53,36 @@ def __init__(self, idx, inputs):
 
         assert idx not in inputs
 
-    def __init_subclass__(cls, **kwargs):
-        super().__init_subclass__(**kwargs)
-        register(cls)
+    # def __init_subclass__(cls: Node, **kwargs: Any) -> None:
```

Comment:
should be commented in before merge

Reply:
there was some problem with this construction, `mypy` complained about it. Will look into this to sort it out.